enterprise_knowledge_retriever/README.md

```diff
@@ -6,327 +6,356 @@
 
-Prompting has a significant effect on the quality of LLM responses. Prompts can be further customized to improve the overall quality of the responses from the LLMs. For example, in this starter kit, the following prompt was used to generate a response from the LLM, where `question` is the user query and `context` is the documents retrieved by the retriever.
+## Experiment with prompt engineering
+
+Prompting has a significant effect on the quality of LLM responses. Customize prompts to improve the overall quality of the responses from the LLMs. For example, with this starter kit, the following prompt uses meta-tags for Llama LLM models and generates a response from the LLM, 
+* `question` is the user query and `context` are the documents retrieved by the retriever.
 ```python
-custom_prompt_template = """[INST]<<SYS>> You are a helpful assistant for question-answering tasks. Use the following pieces of retrieved context to answer the question. If the answer is not in the context, say that you don't know. Cross check if the answer is contained in provided context. If not than say \"I do not have information regarding this\". Do not use images or emojis in your answer. Keep the answer conversational and professional.<</SYS>>
+custom_prompt_template = """<s>[INST] <<SYS>>\nYou're a helpful assistant\n<</SYS>>
+Use the following pieces of context about company annual/quarterly report filing to answer the question at the end. If the answer to the question cant be extracted from given CONTEXT than say I do not have information regarding this.
 
-{context}    
+Context:
+{context}
 
 Question: {question}
 
-Helpful answer: [/INST]"""
-
+Helpful Answer: [/INST]"""
 CUSTOMPROMPT = PromptTemplate(
 template=custom_prompt_template, input_variables=["context", "question"]
 )
 ```
-
-You can make modifications in the following location:
+You can do this modification in the following location:
 ```
-file: prompts/llama7b-knowledge_retriever-custom_qa_prompt.yaml
+file: edgar_qna/prompts
 ```
 
-# Third-party tools and data sources
+## Third-party tools and data sources
 
 All the packages/tools are listed in the requirements.txt file in the project directory. Some of the main packages are listed below:
-
-
 - streamlit (version 1.25.0)
-- langchain (version 0.1.16)
+- llama-hub (version 0.0.25)
+- langchain (version 0.0.266)
+- langchain_community (version 0.0.16)
+- llama-index (version 0.8.20)
 - sentence_transformers (version 2.2.2)
 - instructorembedding (version 1.0.1)
-- chromadb (version 0.4.24)
-- PyPDF2 (version 3.0.1)
-- unstructured_inference (version 0.7.27)
-- unstructured[pdf] (version 0.13.3)
-- PyMuPDF (version 1.23.4)
-- python-dotenv (version 1.0.0)
+- beautifulsoup4 (version 4.12.2)
+- chromadb (version 0.4.8)
+- qdrant-client (version 1.5.2)
+- fastapi (version 0.99.1)
+- unstructured (version 0.8.1)
+- sec-edgar-downloader (version 5.0.2)
+- python-xbrl (version 1.1.1)
+- sseclient (version 0.0.27)
```

Comment:
@renate-snova @snova-amitk, there is an error here, this commit is changing the EKR Readme but the readme proposed changes refer to the EDGAR kit